api-docs/swagger-v1.1.yml

```diff
@@ -602,6 +602,7 @@ paths:
       tags:
         - Usages
       parameters:
+        - $ref: '#/parameters/usages_flat'
```

Comment:
Hij staat hier omdat de relevantie van de `page` en `per_page` opties afhankelijk zijn van deze parameter. Sterker nog, de pagination options zijn 'by default' niet van toepassing; ze zijn dat alleen bij `flat=1`. Het lijkt mij een vrij generieke formatting optie die voor ieder endpoint met hiërarchische data van toepassing kan zijn. Dat dat er maar één is hoeft niet bepalend te zijn (één gedocumenteerde, althans).

Ik heb er overigens ook niet veel moeite mee hem onderaan te zetten hoor. Maar dit waren mijn overwegingen.

Dus: wel of niet onderaan?